### ISSUE TITLE
Console logs and file logs output is not consistent

### ISSUE THREAD

**Issue by sammy007** [nucypher/nucypher#3482]
**Describe the Bug**
The log output to console missing severity and timestamp. Logging output must be consistent.

From the discussion in previous issue related to log output #3480:

> You can file a new issue, but I'm still not sure what you are hoping for. Do you want the log level (INFO, WARN, DEBUG, etc.) for each message to be included in the console output?

Of course, this is how it must work. It's the core purpose of logging for further analysis, anomaly detection and alerting policy. It's very strange to see such liberal treatment of the issue.

**Traceback**

```
2024-04-16T02:06:34+0000 [operator#debug] Cycled teachers; New teacher is (Ursula)⇀SKIPPED↽ (0xSKIPPED)
2024-04-16T02:06:36+0000 [twisted.python.log#info] "IP SKIPPED" - - [16/Apr/2024:02:06:35 +0000] "POST /node_metadata HTTP/1.1" 200 82 "-" "python-requests/2.31.0"
2024-04-16T02:06:38+0000 [operator#debug] Cycled teachers; New teacher is (Ursula)⇀SKIPPED
```

VS console output

```
Cycled teachers; New teacher is (Ursula)⇀SKIPPED↽ (0xSKIPPED)
"IP SKIPPED" - - [16/Apr/2024:02:06:35 +0000] "POST /node_metadata HTTP/1.1" 200 82 "-" "python-requests/2.31.0"
Cycled teachers; New teacher is (Ursula)⇀SKIPPED
```

**System:**
 - OS Version: Docker
 - Nucypher Version: 7.2.0

**Comment by sammy007** [comment]
> > It's very strange to see such liberal treatment of the issue.
> 
> See [#3480 (comment)](https://github.com/nucypher/nucypher/issues/3480#issuecomment-2061149649). I'll say it again here - please tone down the level of conjecture and hyperbole.

Could you focus on issue instead of accusing me of something you totally made up after your arrogant comment that I am "hoping for something". Such dirty tricks does not work here.

**Comment by cygnusv** [comment]
Hey @sammy007, we are looking into it. We suspect it may be something with the way how we use [Twisted](https://github.com/twisted/twisted), the event framework we use internally. See e.g., how we expected formatting should be working:
https://github.com/twisted/twisted/blob/36832d4fb2f2881deaec431b017d9e3b655eb687/src/twisted/logger/_format.py#L358. We'll report back here when we find a solution. 

Having said that, please be aware that we're a small team and we have many open fronts. We're doing our best, with limited resources. Antagonizing the development team is not going to get you far, especially in open source. This is extensive to Threshold's Discord server – our friends giving support, and the community in general, deserve to be treated without sarcastic and abrasive comments.

**Comment by sammy007** [comment]
>  we are looking into it.

Great.

> Having said that, please be aware that we're a small team and we have many open fronts.

I know that, no complains here, why you are stressing that? It's an internet, texting does not carry necessary details so the receiver is free to made them up. It's just a normal technical talk, but I regret if somebody is uncomfortable without a reason.

**Comment by derekpierre** [comment]
Closed via #3483.
